ext/standard/tests/strings/bug74371.phpt

```diff
@@ -0,0 +1,10 @@
+--TEST--
+Bug #74371: strip_tags altering attributes
+--FILE--
+<?php
+
+echo strip_tags('<img src="example.jpg" alt=":> :<">', '<img>');
```

Comment:
Create some additional cases and try to break things.
<img alt=< />
<img alt='foo\' :<' />
etc...

Reply:
Your second example is interesting, the entire tag gets removed (also in the current PHP version) because the `in_q` logic seems to get confused by the `\'`. From what I can find, escaping a quote using backslash is not allowed in XHTML/HTML (https://stackoverflow.com/a/34880976), but the documentation says that broken attributes are not removed.. What do you think, should we leave the current behaviour intact or try to fix that as well here?